integration-tests/src/test/java/org/apache/fineract/integrationtests/CustomSnapshotEventIntegrationTest.java

```diff
@@ -64,7 +64,6 @@ public void testSnapshotEventGenerationWhenLoanInstallmentIsNotPayed() {
             // Create Loan Product
             PostLoanProductsRequest loanProductsRequest = create1InstallmentAmountInMultiplesOf4Period1MonthLongWithInterestAndAmortizationProduct(
                     InterestType.FLAT, AmortizationType.EQUAL_INSTALLMENTS);
-            loanProductsRequest.setEnableInstallmentLevelDelinquency(true);
```

Comment:
Why do you remove these?

Reply:
I found this setting unnecessary as tests were for CustomSnapshot event, delinquency was not getting calculated at any level as delinquency bucket was also not defined...